spec/yaml/swiftnav/sbp/signing.yaml

```diff
@@ -0,0 +1,55 @@
+# Copyright (C) 2018-2022 Swift Navigation Inc.
+# Contact: https://support.swiftnav.com
+#
+# This source is subject to the license found in the file 'LICENSE' which must
+# be be distributed together with this source. All other rights reserved.
+#
+# THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+# EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+# WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+
+package: swiftnav.sbp.signing
+description: Messages relating to signatures
+stable: False
+public: False
+include:
+  - types.yaml
+  - gnss.yaml
+definitions:
+  - MSG_ED25519_SIGNATURE:
+      id: 0x0BD2
+      short_desc: ED25519 signature for groups of RTCM messages
+      fields:
+        - signature:
+            type: array
+            fill: u8
+            size: 64
+            desc: ED25519 signature for messages.
+        - fingerprint:
+            type: array
+            fill: u8
+            size: 20
+            desc: SHA-1 fingerprint of the associated certificate.
+        - signed_messages:
+            type: array
+            fill: u32
+            desc: CRCs of signed messages.
+  - MSG_ED25519_CERTIFICATE:
+      id: 0x0BD3
+      short_desc: ED25519 certificate, split over multiple messages
+      fields:
+        - message_number:
+            type: u8
+            desc: Message number out of total_messages (0 indexed).
+        - total_messages:
```

Comment:
Do we need 2 bytes for this information?  Can we use half a byte?  15*(255-20-1) would allow 3210 bytes for a certificate -- given the Ed25519 cert that I generated manually was about 250 bytes it seems like this should be enough.

Reply:
Could do. As far as I can see, libsbp doesn't have a nice way of encoding integers less than 8 bit though, so I  guess we'd have to do something in the spec like:

```
        - message_num:
            type: u8
            fields:
                  - 7-4:
                      desc: Total number of messages
                          - 0: 0 messages total
                          - 1: 1 messages total
                          - 2: 2 messages total
                          - ...etc..
                  - 0-3:
                      desc: Message number out of total_messages (0 indexed).
                      values:
                          - 0: 1st message
                          - 1: 2nd message
                          - 2: 3rd message
                          - ..etc..
```

Unless there's a nicer way of doing it? Maybe not worth polluting the spec with all that info to save a byte?

Reply:
When we've done this in other messages (MSG_OBS) it's typically just indicated in the docs that the top nibble is the index and the bottom nibble is the count.

Reply:
Have changed to do this in the same way MSG_OBS does it now